sdk/python/localcluster/node.py

```diff
@@ -224,6 +224,7 @@ def setup(self, password: str, config_file: Path, dir: Path, log_tag: str):
             "OTEL_SERVICE_NAME": f"hoprd-{self.p2p_port}",
             "TOKIO_CONSOLE_BIND": f"localhost:{self.tokio_console_port}",
             "HOPRD_NAT": "true" if self.use_nat else "false",
+            "HOPR_CAPTURE_PACKETS": self.dir.joinpath(f"capture_{self.id}.pcap"),
```

Comment:
These files should also be uploaded with the logs? If so, you need to adapt the GHA workflow.